Fuse/Classes/Fuse.swift

```diff
@@ -74,7 +74,7 @@ public class Fuse {
     /// - Returns: A tuple containing pattern metadata
     public func createPattern (from aString: String) -> Pattern? {
         let pattern = self.isCaseSensitive ? aString : aString.lowercased()
-        let len = pattern.characters.count
+        let len = min(pattern.count, maxPatternLength)
```

Comment:
My miss, but the idea is that when the length of the pattern exceeds `maxPatternLength`, it should error out.

Reply:
I can make this change, would you rather it return nil in this case or to throw an error? I suspect returning nil is the better option here as changing the signature of this function to `func createPattern(from aString: String) throws -> Pattern?` would be a breaking change.

Reply:
Fair point. Returning `nil` would be ideal then.